aspnetcore/fundamentals/request-features.md

```diff
@@ -16,7 +16,7 @@ uid: fundamentals/request-features
 
 By [Steve Smith](http://ardalis.com)
 
-Web server features related to how HTTP requests and responses are handled have been factored into interfaces. These interfaces are used by server implementations and middleware to create and modify the application's hosting pipeline.
+Web server features related to how HTTP requests and responses are defined in interfaces. These interfaces are used by server implementations and middleware to create and modify the application's hosting pipeline.
```

Comment:
It feels like there's a word or two missing in this sentence. Maybe you want something like the following:

"Web server implementation details related to HTTP requests and responses are defined in interfaces."